stable/dask/values.yaml

```diff
@@ -0,0 +1,83 @@
+# Default values for dask.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+# nameOverride: dask
+
+scheduler:
+  name: scheduler
+  image: "daskdev/dask"
+  imageTag: "0.17.1"
+  replicas: 1
+  component: "dask-scheduler"
+  serviceType: "LoadBalancer"
+  servicePort: 8786
+  resources: {}
+  aptPackages: >-
+  condaPackages: >-
+  pipPackages: >-
```

Comment:
Should we include these?  These get placed into environment variables that trigger certain actions when using the docker images described above.  If someone were to use their own docker image (that didn't inherit from ours) then these options would have no effect.  

We could keep these for convenience or remove them and resolve this through documentation, telling users how to specify environment varibles.

Reply:
I was wondering about those. Since they are empty by default anyways, you should probably remove them and add a generic way to add env vars.

Reply:
Done.  However I've run into a problem where I want to combine environment variables defined within the template and those that are user provided.  Naively I want the following:

```yaml
          env:
            - name: DASK_SCHEDULER_ADDRESS
              value: {{ template "dask.fullname" . }}-scheduler:{{ .Values.scheduler.servicePort }}
{{ toYaml .Values.jupyter.env | indent 12 }}
```

But this generates the following linting error:

```
$ helm lint .
==> Linting .
[ERROR] templates/dask-jupyter-deployment.yaml: unable to parse YAML
	error converting YAML to JSON: yaml: line 40: could not find expected ':'

Error: 1 chart(s) linted, 1 chart(s) failed
```

Reply:
I suspect that there is a clever way to do with with the helm templating language.  Happy to go through those docs again if that's the right approach.